circle.yml

```diff
@@ -39,10 +39,12 @@ deployment:
   development:
     branch: master
     commands:
-      - python ./db_tasks.py do_migration $DB_CONNECTION_INFO_STAGING
-      - cf login -a https://api.system.staging.digital.gov.au -o dto -u $CF_USER_STAGING -p $CF_PASSWORD_STAGING
+      - python ./db_tasks.py test_migration $CIRCLE_BUILD_NUM $CF_URL_STAGING $CF_EXPORT_URL_STAGING $CF_USER_STAGING $CF_PASSWORD_STAGING $CF_SERVICE_BINDING_STAGING
+      - cf login -a https://api.system.z.cld.gov.au -o dto -u $CF_USER_STAGING -p $CF_PASSWORD_STAGING
       - cf target -o dto -s digital-marketplace
       - cf push -f manifest.yml
+      - cf install-plugin https://github.com/govau/cf-run-and-wait/releases/download/0.1/run-and-wait.linux64
+      - cf run-and-wait dm-dev-api-blue "python ./db_tasks.py do_migration $CIRCLE_BUILD_NUM $DB_CONNECTION_INFO_STAGING"
```

Comment:
I don't understand why the `DB_CONNECTION_INFO` (which contains sensitive credentials) needs to be passed to the task? Shouldn't the task already be able to pick this up from the environment in the same manner as a running application?